test/extended/authorization/per_node_update.go

```diff
@@ -0,0 +1,119 @@
+package authorization
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	o "github.com/onsi/gomega"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceread"
+	authenticationv1 "k8s.io/api/authentication/v1"
+	"k8s.io/apiserver/pkg/authentication/serviceaccount"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/kubernetes/test/e2e/framework/pod"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+	"strings"
+
+	g "github.com/onsi/ginkgo/v2"
+	exutil "github.com/openshift/origin/test/extended/util"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	//go:embed per_node_pod.yaml
+	perNodeCheckPod string
+	//go:embed per_node_rolebinding.yaml
+	perNodeRoleBinding string
+	//go:embed per_node_validatingadmissionpolicy.yaml
+	perNodeCheckValidatingAdmissionPolicy string
+	//go:embed per_node_validatingadmissionpolicybinding.yaml
+	perNodeCheckValidatingAdmissionPolicyBinding string
+)
+
+var _ = g.Describe("[sig-auth][Feature:ServiceAccountTokenNodeBinding][OCPFeatureGate:ValidatingAdmissionPolicy] per-node SA tokens", func() {
```

Comment:
@everettraven example of how to write a test in OCP that relies on ValidatingAdmissionPolicy